### PR TITLE
Aydin first update

### PR DESCRIPTION
**Pull request by aydinsaribudak**

### CODE REVIEW

File: _people/aydin_saribudak.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+collection: people
+permalink: /people/aydins
+position: phd
+title: "PhD Student"
+joined: 2019
+avatar: aydins.jpg
+contacts:
+  email: "aydin.saribudak@rutgers.edu"
+  office: "CoRE 707"
+  [scholar][scholarlink]
```
Comment (iparask) [inline _people/aydin_saribudak.md:12]: This line produces the following error:
```
Error: YAML Exception reading /Users/iparask/radical-group/_people/aydin_saribudak.md: (<unknown>): could not find expected ':' while scanning a simple key at line 12 column 3
                    done in 0.
```

As a result your page does not get created. Check [here](https://raw.githubusercontent.com/radical-group/radical-group.github.io/master/_people/ioannis_par.md) to see how to incorporate a Google scholar link

File: _people/aydin_saribudak.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+joined: 2019
+avatar: aydins.jpg
+contacts:
+  email: "aydin.saribudak@rutgers.edu"
+  office: "CoRE 707"
+  [scholar][scholarlink] 
+  github: https://github.com/aydinsaribudak
+  linkedin: https://www.linkedin.com/in/aydin-saribudak-1113b927
+  researchgate: https://www.researchgate.net/profile/Aydin_Saribudak
+[scholar]: https://scholar.google.com/citations?user=Cx_5JNoAAAAJ&hl=en&oi=ao
```
Comment (iparask) [inline _people/aydin_saribudak.md]: This should be inside the contacts part of the page. See above comment.

**Review by iparask — CHANGES_REQUESTED** [review]

File: _people/aydin_saribudak.md (hunk 2 above)
Comment (aydinsaribudak) [reply _people/aydin_saribudak.md]: I update the file as requested. I make the "scholar' inside the contacts.

**Comment by iparask** [comment]
Outdated form #115

**Pull request closed without merging**